versions/v6/templates/story/full-width-story.php

```diff
@@ -0,0 +1,17 @@
+<?php disallow_direct_load('full-width-story.php');?>
+<?php add_filter('the_content', 'kill_empty_p_tags', 999); ?>
+<?php
+$display_header = get_field( 'story_fw_display_standard_header' );
+?>
```

Comment:
I know we're not always consistent about this, but it might be nice when we have multiple lines like this just to have them under a single `<?php ?>` tag, to make it more readable.

```suggestion
<?php
disallow_direct_load('full-width-story.php');
add_filter('the_content', 'kill_empty_p_tags', 999);
$display_header = get_field( 'story_fw_display_standard_header' );
?>
```